charts/nginx/templates/nginx-service.yaml

```diff
@@ -41,11 +41,11 @@ spec:
     component: ingress-controller
     release: {{ .Release.Name }}
   ports:
-    - port: {{ .Values.ports.http }}
+    - port: 80
```

Comment:
it is only supported to use load balancer with external port 80, 443 due to CORS. However, it is not necessary for the external port to match the port used by the container, so we hard code here what is required and change the default nginx container ports to non privileged ports.